docs/content/en/usage.md

```diff
@@ -3,35 +3,91 @@ title: Usage
 description: 'Access text, images, and other media with Nuxt and the Sanity headless CMS.'
 category: Getting started
 position: 4
-version: 0.39
+version: 0.40
 ---
 
 This module globally injects a `$sanity` helper, meaning that you can access it anywhere using `this.$sanity`. For plugins, asyncData, fetch, nuxtServerInit and middleware, you can access it from `context.$sanity`.
 
 ## Reference
 
-### `fetch`
+### Example with `asyncData`
 
-This enables you to perform a GROQ query against your Sanity dataset. By default it returns a `Promise<unknown>` although you can customise the type of the return.
```

Comment:
This is meant to be a section explaining how `$sanity.fetch` works - could we keep this content and move the examples to the end (or create a code block with all four)?

Reply:
oh I see. I thought about "fetch" from Nuxt and not from this library😅

Reply:
The indication "By default it returns a `Promise<unknown>` although you can customise the type of the return." only for TypeScript users, right?

Reply:
yes, the bit about customing the type is